migrate/apiVersion/migrate.go

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package migrate
+package apiVersion
```

Comment:
I wouldn't rename this package. I don't think "apiVersion" explains the functionality. I would rather keep it in the top level as it was before and just put the SQL migration to a separate package called "database" so it becomes "migrate/database"